server/index.js

```diff
@@ -10,6 +10,14 @@ const sessionStore = new SequelizeStore({db})
 const PORT = process.env.PORT || 8080
 const app = express()
 const socketio = require('socket.io')
+const fileUpload = require('express-fileupload')
+const cloudinary = require('cloudinary').v2;
+
+cloudinary.config({
+  cloud_name: 'ddfklmpvy',
+  api_key: '953796559747358',
+  api_secret: 'jXCsVgk7-OXs-Z__P5eH97exTSU'
```

Comment:
We'll probably get a warning about exposed secrets on this one, might be okay short term for the presentation but we should think about adding a file to .gitignore and importing it here, we'll also have to set these as config vars on Heroku to get this to work on the deployed site